### PR TITLE
Add DNApi

### PR DESCRIPTION
**Pull request by kepbod**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/dnapi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  name: dnapi
+  version: "1.1"
+
+build:
+  number: 0
+  skip: True  # [py27]
+
+source:
+  fn: master.zip
+  url: https://github.com/jnktsj/DNApi/archive/master.zip
```
Comment (bgruening) [inline recipes/dnapi/meta.yaml]: please use a specific commit or version, not master

Comment (kepbod) [inline recipes/dnapi/meta.yaml]: OK, I close this RP.

Comment (bgruening) [inline recipes/dnapi/meta.yaml]: You don't need to close it. Just push to this branch and it will be updated.
Otherwise this PR looks good! :)

Comment (kepbod) [inline recipes/dnapi/meta.yaml]: I have created an new RP with specific version. So we could discard this RP.

**Comment by bgruening** [comment]
replaced by: https://github.com/bioconda/bioconda-recipes/pull/2975

**Comment by kepbod** [comment]
Thx!!!

**Pull request closed without merging**